docs/api/ref/Errors.md

```diff
@@ -36,6 +36,22 @@ Contains constants representing the errors that can be encountered when calling
 * [`DesktopAgent.findIntent`](DesktopAgent#findintent)
 * [`DesktopAgent.findIntentsByContext`](DesktopAgent#findintentsbycontext)
 
+## `ResultError`
+
+```typescript
+enum ResultError {
+  NoResultReturned = 'NoResultReturned',
```

Comment:
Should this be an error or should `getResult` just return `Promise<void>`

Reply:
I suspect that, if a result was expected, the promise rejection is (slightly) more useful as it can be handled with other errors, rather than requiring a check for an undefined result. However, it works as a pattern either way.